spec/models/user_spec.rb

```diff
@@ -367,4 +366,21 @@
       it { expect(subject.admin?).to be_truthy }
     end
   end
+
+  describe '#full_name' do
+    subject { create(:user, first_name: 'first', last_name: 'last') }
+
+    it 'should compose #first_name' do
+      expect(subject.first_name).to eq('first')
+    end
+    it 'should compose #last_name' do
+      expect(subject.last_name).to eq('last')
+    end
+    it 'should return "first_name last_name"' do
+      expect(subject.full_name).to eq('first last')
+    end
```

Comment:
@matthew-campbell - all of these are good tests but this section should really only be testing the subject method - which this specific test does adequately.

See comments for the other tests.